app/components/frame-viewer.spec.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import assert from 'assert';
+import { mount } from 'enzyme';
+import FrameViewer from './frame-viewer';
+import FrameAnnotator from '../classifier/frame-annotator';
+
+const subject = {
+  locations: [{ image: 'www.testimage.com' }]
```

Comment:
Maybe you can leverage the images from the mock data from the dev classifier instead of using a fake url?